starlark/Cargo.toml

```diff
@@ -26,7 +26,7 @@ codemap = "0.1.1"
 codemap-diagnostic = "0.1.1"
 lalrpop-util = "0.16.0"
 linked-hash-map = "0.5.1"
-linked_hash_set = { version = "0.1.3", optional = true }
+linked_hash_set = "0.1.3"
```

Comment:
This is increasing the number of mandatory dependency. This is why it was a compilation option. I'd rather not increase it if we can avoid it.